chef_master/source/server_backup_restore.rst

```diff
@@ -124,6 +127,75 @@ When restoring Chef server data, the previously backed-up files will be required
 
       $ chef-manage-ctl reconfigure
 
+
+Using Chef Backend
+=====================================================
+
+The backup and restore process allows you to restore a data backup
+into a newly built cluster in a disaster recovery scenario. It is not
+currently intended for the recovery of an individual machine in the
+chef-backend cluster or for point-in-time rollback of an existing
+cluster.
+
+Backup
+------
+
+To be able restore your data in the case of an emergency, you need
+backups of
+
+- the data in your Chef backend cluster
+- the configuration from your Chef server
+
+1. On a follower chef-backend node, run ``chef-backend-ctl backup``
+2. On a chef-server node run: ``chef-server-ctl backup --config-only``
+3. Move the tar archives created in steps (1) and (2) to a long-term
+   storage location.
+
+Restore
+-------
+
+To restore a Chef Backend-based Chef server cluster:
+
+1. On the first machine that you want to use in your new Chef Backend
+   cluster. The argument to the ``--publish_address`` option should be
+   an IP address that can be used to reach the node you are restoring.
+
+.. code-block:: bash
+
+   chef-backend-ctl restore --publish_address X.Y.Z.W /path/to/backup.tar.gz
+
+2. Join additional nodes to your Chef Backend cluster (note: if you
+   are simply trying to test and verify your restore process, you can
+   skip the additional backend nodes)
```

Comment:
By 'skip the additional backend nodes', do you mean that for testing the restore process the user should:
1. Install the first machine in the backend cluster and 0 additional nodes.
2. Install the first machine in the backend cluster and 1 backend node.